hooks/useWalletProvider.tsx

```diff
@@ -100,11 +100,11 @@ const useWalletProvider = () => {
       // modal, as "User closed modal"
       console.log('error', e)
     }
-  }, [handleAccountsChanged, web3Modal])
+  }, [])
```

Comment:
Similarly -- What's the purpose of removing this for this repo? Seems to still exist on `main` in the example chat:

https://github.com/xmtp/example-chat-react/blob/f0955005aa0b94b50eb236436c4e729c46f6286e/hooks/useWalletProvider.tsx#L103